llvm/docs/DeveloperPolicy.rst

```diff
@@ -86,6 +86,11 @@ that notices of confidentiality or non-disclosure cannot be respected.
 Making and Submitting a Patch
 -----------------------------
 
+The LLVM community requires contributors to have a public email address
```

Comment:
Should this be the first paragraph of the section? How about putting it after the non-disclosure notice paragraph, or immediately before?